client/src/components/Home/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from 'antd';
+import './style.css';
+
+function Home() {
+  return (
+    <section className='home'>
+      <h1 className='home--heading'>
+        Lorem ipsum dolor sit amet, consectetuer adipiscing elit. 
+        Aenean commodo
+      </h1>
+      <div className='home--buttons-container'>
```

Comment:
this is not BEM 
the class should be (home--(Subscription, Visitors or any word will meaningful)).
and the child will be (home--(Subscription, Visitors or any word will meaningful)-button)